Airport.js

```diff
@@ -91,9 +77,9 @@ class Airport {
         return this.planes;
     }
 
-    static print(planes) {
-        return JSON.stringify(planes);
+    print(providedPlanes) {
```

Comment:
Please remove this method to a separate function OUT of the `Airport` class.

Reply:
moved to the separate folder "helpers"